### PR TITLE
style(ui): web ui refinement

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

web ui refinement

1. Add IconButton to support desktop icon button hover and press style
image

2. Add border-radius and shadow to desktop main content card
3. Adjust ACModal border-radius
4.  Create Account link should not stick at buttom on desktop
image

## Linear Issue Reference

log-2503
log-2560
log-2488

## Testing

test locally
@logto-io/eng

### CODE REVIEW

File: packages/ui/src/scss/_desktop.scss [hunk 1]
```diff
@@ -65,6 +65,7 @@ $font-family: -apple-system,
   --color-toast: #34353f;
   --color-overlay: rgba(0, 0, 0, 30%);
   // shadows
+  --shadow-1: 0 8px 16px rgba(66, 41, 159, 4%);
```
Comment (demonzoo) [inline packages/ui/src/scss/_desktop.scss:68]: `--shadow-1` is updated to `0 4px 8px rgba(0, 0, 0, 8%);`

**Comment by demonzoo** [comment]
> > The primary button background color doesn't seem to be right. Please check latest design.
> > Should be `primary-70` (#7958ff) in dark theme
> 
> That'a a random color I set in AC settings.

Oh, is it? OK then

**Review by demonzoo — APPROVED** [review]

File: packages/ui/src/scss/_desktop.scss (hunk 1 above)
Comment (demonzoo) [reply packages/ui/src/scss/_desktop.scss:68]: Sure, approved in prior. Update the shadow styles and it's good to merge

Comment (simeng-li) [reply packages/ui/src/scss/_desktop.scss]: checked updated.

**Review by wangsijie — APPROVED** [review]

**Pull request merged**